packages/ubuntu_desktop_installer/lib/l10n/app_en_US.arb

```diff
@@ -120,6 +120,28 @@
   "installationTypeManual": "Something else",
   "installationTypeManualInfo": "You can create or resize partitions yourself, or choose multiple partitions for Ubuntu",
 
+  "selectGuidedStoragePageTitle": "Erase disk and install Ubuntu",
+  "selectGuidedStorageDropdownLabel": "Select drive:",
+  "selectGuidedStorageDropdownItem": "{path} - {size} {name}",
+  "@selectGuidedStorageDropdownItem": {
+    "description": "The description of a storage that can be selected for guided partition",
+    "placeholders": {
+      "path": {},
+      "size": {},
+      "name": {}
```

Comment:
It would be useful to have descriptions of those parameters to help translators.

Reply:
let's apply the appropriate formatting here and elsewhere when we actually have the names etc. :)